lib/utils/html-tag.html

```diff
@@ -12,13 +12,31 @@
   (function() {
     'use strict';
 
+    class HTMLLiteral {
+      constructor(string) {
+        this.innerHTML = string;
+      }
```

Comment:
Consider adding a `toString()` implementation with the literal value

Reply:
Done!